nextest/runner/src/runner.rs

```diff
@@ -314,9 +315,20 @@ impl<'list> TestRunner<'list> {
 
         let handle = cmd.start()?;
 
-        // TODO: timeout/kill logic
+        let now = Instant::now();
 
-        let output = handle.into_output()?;
+        let output = loop {
+            let result = handle.try_wait();
+            println!("{:?} elapsed time is {}", &test, now.elapsed().as_secs());
+            if let Ok(None) = result {
+                sleep(Duration::new(2, 0));
```

Comment:
Do we want to up this limit?  For reference, there are ~100 tests take over 20sec to complete (see histogram https://ci-artifacts.diem.com/testhistory/diem/diem/auto/ci-test.yml/1091973229/report/index.html#graph).

Reply:
huh somehow got an older revision on this.  will update.

Reply:
resolved